app/src/main/java/io/apicurio/registry/rest/v2/UsersResourceImpl.java

```diff
@@ -65,7 +65,7 @@
     public UserInfo getCurrentUserInfo() {
         UserInfo info = new UserInfo();
         info.setUsername(securityIdentity.getPrincipal().getName());
-        info.setDisplayName(securityIdentity.getPrincipal().getName()); // TODO need a better implementation of this
+        info.setDisplayName(securityIdentity.getPrincipal().getName()); // TODO need a better implementation of this, maybe use claims first_name and last_name
```

Comment:
Yes this makes sense.

Reply:
I think that, when present, `preferred_username` claim is probably a better idea.